tls/handshake_client.go

```diff
@@ -513,6 +513,9 @@ func (c *Conn) clientHandshake() error {
 		}
 	} else {
 		if err := hs.doFullHandshake(); err != nil {
+            if err == ErrCertsOnly {
+                c.sendAlert(alertCloseNotify)
+            }
```

Comment:
Indentation if off here.

Reply:
Sorry about that.  My typical tabstop and shiftwidth are interacting with this project in a kooky way where things look good to me in vi but are actually messed up.  I believe this is fixed in https://github.com/zmap/zcrypto/pull/100/commits/45bd5336e1b95a543b30e7c71200eacf30e8d0b4